File: Daily problems/Solution 17-02-2026/shubham/q1.java


public class q1 {
    public static void main(String[] args) {
       int n =19;
       int sum=0;
       while(n>0){
        int digit=n%10;
        sum=sum+(digit*digit);
        n=n/10;

       }
       System.out.println(sum);
    }
}

File: Daily problems/Solution 17-02-2026/shaurya/q1.cpp
class Solution {
public:
    bool isHappy(int n) {
        int sum=0;
        while(n>0){
            int r=n%10;
            sum+=r*r;
            n/=10;
        
        }
        if(sum==1){
            return true;
        }
        if(sum==4){
            return false;
        }
        
    
      return  isHappy(sum);
    }
};